utils/rpms/mercury.sh

```diff
@@ -36,14 +37,17 @@ list_files files "${SL_MERCURY_PREFIX}/lib64/lib*.so.*"
 clean_bin "${files[@]}"
 append_install_list "${files[@]}"
 
+ARCH="${isa}"
+build_package "mercury"
+
 TARGET_PATH="${libdir}/mercury"
 list_files files "${SL_MERCURY_PREFIX}/lib64/mercury/libna_plugin_ofi.so"
 clean_bin "${files[@]}"
 append_install_list "${files[@]}"
 
 ARCH="${isa}"
-DEPENDS=("${libfabric_lib} >= ${libfabric_version}")
-build_package "mercury"
+DEPENDS=("${libfabric_lib} >= 1.20")
```

Comment:
This says 1.20, but package_info.sh still has 1.22?

Reply:
right, this is a different version, maybe I could define a new env that says min_version instead. The min version is different from the version that we ship. You want to be able to give some room when generating rpms for folks that do not necessarily have the latest libfabric (and it's not always possible to use the one that we ship).